### PR TITLE
Feature/768 workaround for celo

### PR DESCRIPTION
**Pull request by testing-life**
for celo users, the result should be:
image

for others, the result should be 
image

also, fixed token symbol not appearing in thank you page
image

### CODE REVIEW

File: src/newLaunch/stages/stage8.html [hunk 1]
```diff
@@ -1,63 +1,91 @@
...
 
           will have to fund the ${wizardState.launchTypeTitle} with
 
           <span if.bind="wizardState.requiredProjectTokenDeposit">
             <formatted-number
               value.to-view="wizardState.requiredProjectTokenDeposit  | ethwei:launchConfig.tokenDetails.projectTokenInfo.decimals"
-              thousands-separated></formatted-number>
+              thousands-separated
+            ></formatted-number>
             ${launchConfig.tokenDetails.projectTokenInfo.symbol}
```
Comment (giladt) [inline src/newLaunch/stages/stage8.html:56]: The symbol is not shown. Can you check that?
image

File: src/newLaunch/stages/stage8.ts [hunk 2]
```diff
@@ -24,4 +27,10 @@ export class Stage8 extends BaseStage<ILaunchConfig> {
   get ipfsURL(): string {
     return this.ipfsService.getIpfsUrl(this.wizardState.launchHash);
   }
+
+  attached():void{
+    if (EthereumService.targetedNetwork === Networks.Celo || EthereumService.targetedNetwork === Networks.Alfajores) {
```
Comment (giladt) [inline src/newLaunch/stages/stage8.ts:32]: Can use `isCeloLikeNetwork`?

**Review by giladt — CHANGES_REQUESTED** [review]

File: src/newLaunch/stages/stage8.html (hunk 1 above)
Comment (testing-life) [reply src/newLaunch/stages/stage8.html:56]: hm so with D2D it's correct. i'll check rinkeby with GILAD
image

File: src/newLaunch/stages/stage8.html [hunk 3]
```diff
@@ -1,63 +1,91 @@
...
-          appear in and be used in the Prime Launch interface. The approval takes place in the <a
-            href.bind="multiSigWalletUri" target="_blank" rel="noopener noreferrer">Prime Launch multisig wallet</a>.
-          Your ${wizardState.launchTypeTitle} may take up to 5 minutes to appear in the multisig. Reviews generally take
-          5-7 days."</p>
+        <p if.to-view="dataToken">
+          <span class="importantText"
+            >Note that when you generate a new token, the value will only be displayed once, so be sure to write it
+            down</span
+          >
+          <span class="dataTokenWrapper">${dataToken}</span>
```
Comment (hiaux0) [inline src/newLaunch/stages/stage8.html:21]: I left a comment about html wrap formatter from vscode.
Can you check if that's the case for you?

Ideally, not major formatting should happen

**Review by hiaux0 — APPROVED** [review]
the html formatting really makes it hard to spot the actual changes

File: src/newLaunch/stages/stage8.html (hunk 3 above)
Comment (testing-life) [reply src/newLaunch/stages/stage8.html:21]: that's the correct setting, no ?
image

**Review by giladt — DISMISSED** [review]

File: src/newLaunch/stages/stage8.html (hunk 3 above)
Comment (hiaux0) [reply src/newLaunch/stages/stage8.html:21]: yeah, then I don't know why you have all those html attribute changes?

**Review by hiaux0 — DISMISSED** [review]

**Pull request merged**